packages/taco/src/conditions/schemas/common.ts

```diff
@@ -23,10 +20,11 @@ export const plainStringSchema = z
     `Any string that is not a Context Parameter i.e. does not start with \`${CONTEXT_PARAM_PREFIX}\`.`,
   );
 
-export const UserAddressSchema = z.enum([
-  USER_ADDRESS_PARAM_DEFAULT,
-  USER_ADDRESS_PARAM_EXTERNAL_EIP4361,
-]);
+export const UserAddressSchema = z
+  .literal(USER_ADDRESS_PARAM_DEFAULT)
+  .describe(
+    'This is a context parameter that will be replaced at decryption time. It represents the Ethereum address of the user attempting decryption.',
```

Comment:
context variable - https://docs.taco.build/authentication/conditioncontext-and-context-variables#context-variables.

```suggestion
    'This is a context variable that will be replaced at decryption time. It represents the Ethereum address of the requester attempting decryption.',
```